app/views/components/linkAsButton.scala.html

```diff
@@ -22,6 +22,6 @@
 id: Option[String] = None,
 )(implicit messages: Messages)
 
-<a href="@link" role="button" class="govuk-button govuk-button" @id.map(x => s"id=$x")>
+<a href="@link" role="button" class="govuk-button govuk-button" data-module="govuk-button" draggable="false" data-prevent-double-click="true" @id.map(x => s"id=$x")>
```

Comment:
@gastonracui just reading more about data-prevent-double-click="true", I don't think it's needed here as there no risk on submitting data twice (it just a continue button without any submission data). Let's remove this.